test/unit/model/qti/CustomInteractionAsset/CustomInteractionAssetExtractorAllocatorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoQtiItem\test\unit\model\qti\CustomInteractionAsset;
+
+use oat\generis\test\TestCase;
+use oat\taoQtiItem\model\qti\CustomInteractionAsset\CustomInteractionAssetExtractorAllocator;
+use oat\taoQtiItem\model\qti\CustomInteractionAsset\Extractor\NullAssetExtractor;
+use oat\taoQtiItem\model\qti\CustomInteractionAsset\Extractor\TextReaderAssetExtractor;
+use Ramsey\Uuid\Uuid;
```

Comment:
I don't think this particular extension requires that lib by default.
Please either
1. Require it as a `dev` dependency
2. Use something like `uniqid()` instead, which should be sufficient for the tests.

Reply:
done